Animations/src/renderer/Renderer.cpp

```diff
@@ -351,6 +351,8 @@ namespace MathAnim
 			shader3DTransparent.compile("assets/shaders/shader3DTransparent.glsl");
 			shader3DComposite.compile("assets/shaders/shader3DComposite.glsl");
 			pickingOutlineShader.compile("assets/shaders/pickingOutline.glsl");
+#else
+#error Neither _DEBUG nor _RELEASE are defined
```

Comment:
Good call here. I need to use compile assertions more often for cases like this :)